### ISSUE TITLE
Error handling over devp2p

### ISSUE THREAD

**Issue by jflo** [hyperledger/besu#5814]
```2023-08-25 16:15:03.670+00:00 | EthScheduler-Workers-24175 | ERROR | Besu | Uncaught exception in thread "EthScheduler-Workers-24175"
java.lang.IllegalArgumentException: Value -1 cannot be represented as an unsigned int (it is negative or too big)
        at org.apache.tuweni.bytes.Checks.checkArgument(Utils.java:36)
        at org.apache.tuweni.bytes.Bytes.ofUnsignedInt(Bytes.java:413)
        at org.apache.tuweni.bytes.Bytes.ofUnsignedInt(Bytes.java:400)
        at org.hyperledger.besu.ethereum.rlp.RLPOutput.lambda$writeUnsignedInt$2(RLPOutput.java:222)
        at org.hyperledger.besu.ethereum.rlp.RLPOutput.processZeroByte(RLPOutput.java:315)
        at org.hyperledger.besu.ethereum.rlp.RLPOutput.writeUnsignedInt(RLPOutput.java:222)
        at org.hyperledger.besu.ethereum.eth.encoding.TransactionAnnouncementEncoder.lambda$encodeForEth68$1(TransactionAnnouncementEncoder.java:115)
        at org.hyperledger.besu.ethereum.rlp.RLPOutput.writeList(RLPOutput.java:256)
        at org.hyperledger.besu.ethereum.eth.encoding.TransactionAnnouncementEncoder.encodeForEth68(TransactionAnnouncementEncoder.java:115)
        at org.hyperledger.besu.ethereum.eth.encoding.TransactionAnnouncementEncoder.encodeForEth68(TransactionAnnouncementEncoder.java:89)
        at org.hyperledger.besu.ethereum.eth.messages.NewPooledTransactionHashesMessage.create(NewPooledTransactionHashesMessage.java:53)
        at org.hyperledger.besu.ethereum.eth.transactions.NewPooledTransactionHashesMessageSender.sendTransactionHashesToPeer(NewPooledTransactionHashesMessageSender.java:60)
        at org.hyperledger.besu.ethereum.eth.transactions.TransactionBroadcaster.lambda$sendTransactionHashes$7(TransactionBroadcaster.java:183)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1589)```


This seems like p2p noise that should be handled better.

**Comment by macfarla** [comment]
PR #5826

**Comment by macfarla** [comment]
fixed by #5826
